sdk/include/ddk/scsi.h

```diff
@@ -3136,7 +3136,7 @@ typedef struct _CDVD_CAPABILITIES_PAGE {
   UCHAR Reserved6:1;
   UCHAR LoadingMechanismType:3;
   UCHAR SeparateVolume:1;
-  UCHAR SeperateChannelMute:1;
+  UCHAR SeparateChannelMute:1;
```

Comment:
This seems to be the "official" typoed name:
https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/shared/scsi.h#L4898

Reply:
Committed.